homeassistant/components/light/demo.py

```diff
@@ -53,61 +56,66 @@ def __init__(
         self._effect = effect
 
     @property
-    def should_poll(self):
+    def should_poll(self) -> bool:
         """No polling needed for a demo light."""
         return False
 
     @property
-    def name(self):
+    def name(self) -> str:
         """Return the name of the light if any."""
         return self._name
 
     @property
-    def brightness(self):
+    def available(self) -> bool:
```

Comment:
How about a trade-off of keeping it in and just adding a comment, that please support if possible? It allows much nicer display of disconnected devices (in case of bt) than not finding what you are looking for & grepping the logs.